modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container to restrict specific container activities and enhance security.
+
+[INFO]
+====
+To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+====
+
+.Procedure
+
+. Use the public https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile] image based on Red Hat UBI.
```

Comment:
Can you specify how I should use it? Is there a command?

Reply:
Also, "use" doesn't make clear what it's being used for, can this be reworded to explain how / what it's used for?

Reply:
Need SME input here.

Reply:
@chmouel I've asked you for clarification in the google doc here: https://docs.google.com/document/d/1swGqQRF4kPg9EsMue9I-XYI654oo0FMtwP0cROFU3so/edit?disco=AAAAbLSIAtE